pkgs/development/libraries/glibc/default.nix

```
@@ -14,16 +14,19 @@ let
   ];
 in
 
-callPackage ./common.nix { inherit stdenv; } {
+((lib.makeOverridable (callPackage ./common.nix { inherit stdenv; }) {
+  inherit withLinuxHeaders withGd;
+}).override(prev: {
```

```suggestion
}).override({
```

override overrides the function args not the attrs

---

Ah, thanks for catching that.